troubleshooting/nvidia-troubleshooting.md

```diff
@@ -71,3 +78,70 @@ If you see a terminal window in the top left corner, it means you most likely su
 
 1.  Reboot and let the system boot from the normal boot configuration. You should be able to use X under Xen now.
 
+
+Disabling Nouveau
+---------------------
```

Comment:
@marmarek 
 I've added a section giving a fair mention that this disables nvidia support all together.

The nice thing about this patch it that editing the GRUB boot config at boot is forgiving, so if you disable nouveau all together and weird stuff happens then on reboot it will boot with nouveau enabled again.